Dockerfile

```diff
@@ -125,6 +125,7 @@ WORKDIR /workspace/machines
 # Download old WASM module roots
 #RUN bash -c 'mkdir 0x21f708e444c3afb7689fa5d0737b3942fd19012c0081d359ba3d59b7643d7810 && cd $_ && wget https://github.com/OffchainLabs/nitro/releases/download/devnet-consensus-v1/machine.wavm.br'
 RUN bash -c 'mkdir 0xb7905959ec167e0777bbbd6c339b0c98d676729cb502722aa01a34964f817ca3 && ln -s $_ latest && cd $_ && wget https://github.com/OffchainLabs/nitro/releases/download/devnet-consensus-v2/machine.wavm.br'
```

Comment:
We should remove the link to latest here

Reply:
I've changed the scheme a bit to just override the symlink to latest each time. Alternatively, we can keep moving where the latest symlink is in the list of commands, or explicitly symlink one of them after the downloads. Let me know what you think is best.